k8s/overlays/demo/web-deployment-redis-patch.yaml

```
@@ -0,0 +1,21 @@
+        securityContext:
+          runAsNonRoot: true
+          runAsUser: 1000
+          runAsGroup: 1000
+          allowPrivilegeEscalation: false
+        env:
+        # Add REDIS_URL from configmap
+        - name: REDIS_URL
+          valueFrom:
+            configMapKeyRef:
+              name: django-config
+              key: REDIS_URL
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)